src/window/resource_settings.c

```diff
@@ -180,7 +185,7 @@ static void handle_input(const mouse *m, const hotkeys *h)
         return;
     }
     if (input_go_back_requested(m, h)) {
-        window_advisors_show();
+        window_go_back();
```

Comment:
We can use the more generic `go-back` process instead of showing the advisor dialog. This seems to work perfectly fine in my testing and allows me to return to the empire screen instead

Reply:
Yeah, this is the correct way to do it :wink: